assets/src/edit-story/components/header/buttons/publish.js

```diff
@@ -88,14 +96,23 @@ function Publish() {
     ? __('Schedule', 'web-stories')
     : __('Publish', 'web-stories');
 
+  const button = (
+    <Primary
+      onMouseEnter={refreshChecklist}
+      onClick={handlePublish}
+      isDisabled={!capabilities?.hasPublishAction || isSaving || isUploading}
+    >
+      {text}
+    </Primary>
+  );
+
+  const wrappedWithTooltip = (
+    <WithTooltip title={tooltip}>{button}</WithTooltip>
+  );
```

Comment:
This tooltip should not appear (and the check shouldn't run) when the feature is not enabled.

<img width="541" alt="Screenshot 2020-11-20 at 16 30 57" src="https://user-images.githubusercontent.com/841956/99817872-c5722380-2b4d-11eb-99f9-ed9a706624ef.png">